GoogleForCreators/web-stories-wp
# Remove DEPRECATED_THEME from Dashboard

## Context
Replace all the deprecated theme usage in the Dashboard code with colors and utils from the design system.



## Summary
- Remove unused components and code where I could find them and be sure they were unused from the dashboard only
- Replace `DEPRECATED_THEME` usage in the dashboard with the design-system theme from the provider and remove DEPRECATED_THEME entirely from the dashboard's theme provider.


## Relevant Technical Choices
-  relying somewhat on integration tests to discover issues with removed code as described in the ticket, also encountered no errors during my testing. used "find all references" in VS Code to do unused code discovery in index files.
- where there were no design references available (i.e. figma) I used the property in the design system theme that seemed most reasonable.


## To-do



## User-facing changes
- before/after shown in the comments


## Testing Instructions
- Relying on regression to ensure nothing is broken from removing unused code. 
- To manually test the dashboard for regression (may not be an exhaustive list, if my reviewers could point out any holes in my testing methodology it would be greatly appreciated):
- Go to the dashboard, create, duplicate, delete, rename, etc. stories
- Switch between list and grid views
- Make the viewport smaller and test the nav menu button
- Explore templates and view their details
- Enable preview stories experiment and preview stories
- Use the keyboard to navigate the dashboard


### QA






This PR can be tested by following these steps:

1. See Testing instructions

### UAT








## Reviews

### Does this PR have a security-related impact?
no


### Does this PR change what data or activity we track or use?
no


### Does this PR have a legal-related impact?
no


## Checklist




---



Fixes #6707
